go/engine/kex2_provisioner.go

```diff
@@ -295,7 +295,8 @@ func (e *Kex2Provisioner) CounterSign2(input keybase1.Hello2Res) (output keybase
 	if len(string(input.DeviceEkKID)) != 0 && userEKBoxStorage != nil {
 		output.UserEkBox, err = e.makeUserEKBox(m, input.DeviceEkKID)
 		if err != nil {
-			return output, err
+			output.UserEkBox = nil
```

Comment:
I would name this error something other than `err`, since the `m.CTrace` is going to report on what this value is, it might get confusing.  How about `w` or `tmp` if it's a warning?

Reply:
makes sense, fixed!